Jenkinsfile

```diff
@@ -385,6 +385,60 @@ pipeline {
                         }
                     }
                 }
+             stage('cart-master-nightly') {
+                when { branch 'master' }
+                agent {
+                    label 'ci_vm1'
+                }
+                options {
+                    timeout(time: 60, unit: 'MINUTES')
+                }
+                steps {
+                    provisionNodes NODELIST: env.NODELIST,
+                        node_count: 1,
+                        snapshot: true
+                    runTest stashes: [ 'CentOS-install', 'CentOS-build-vars' ],
```

Comment:
This tells Jenkins which artifacts to copy in (therefore what to test), so should be 

                        runTest stashes: [ 'CentOS-master-install', 'CentOS-master-build-vars' ],